tests/test_flats.py

```diff
@@ -153,6 +164,57 @@ def test_treerings_flat(self):
         assert cov01 > 0.5*tot_counts
         assert cov11 > 0.5*tot_counts
 
+    def test_sed_flat(self):
+        """Test LSST_Flat with an sed item"""
+
+        counts_per_iter = 100
+        niter = 5
+        tot_counts = counts_per_iter * niter
+        # Use an SED with a tight wavelength range in the z band
```

Comment:
Do you mean g band here?

Reply:
Thanks.  I originally did something in z band I think, but I seem to have ended up in g band.  I think because even in z, there were a fair number of photons falling through the bottom, so here that's not an issue.